client/src/actions/index.js

```diff
@@ -27,21 +28,43 @@ export const signOut = () => {
 
 export const createStream = formValues => async (dispatch, getState) => {
   const { userId } = getState().auth;
-  const payload = {
-    description: formValues.description,
-    title: formValues.title,
-    userid: userId
-  };
+  let streamLink;
   database
-    .collection("streams")
-    .doc(userId)
-    .set(payload)
-    .then(() => {
-      dispatch({ type: CREATE_STREAM, payload: payload });
-      history.push("/");
+    .collection("streamlinks")
```

Comment:
I approved this since it works, but I'm kinda confused by this on second thought. I dont even see a streamlinks collection in firebase.